blocks/url-input/index.js

```diff
@@ -215,7 +215,7 @@ class UrlInput extends Component {
 								onClick={ () => this.props.onChange( post.link ) }
 								aria-selected={ index === selectedSuggestion }
 							>
-								{ post.title.rendered }
+								{ post.title.rendered || `( ${ __( 'no title' ) } )` }
```

Comment:
Maybe a bit simpler like this: `post.title.rendered || __( 'no title' )`

Reply:
Sure, I can do that. I added the brackets so they are more in line with how no title posts are displayed in other places in WP admin as @afercia noted.

Reply:
Oh sorry for the confusion, I see what you mean now and you're right we should add the parenthesis. `post.title.rendered || __( '(no title)' )` 

A quick search in the WP source code shows that the parenthesis are included in the translated message.